### PR TITLE
[zh-CN]: update function and class available to worker

### PR DESCRIPTION
**Pull request by ChaominRuan**

### CODE REVIEW

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 1]
```diff
@@ -3,19 +3,27 @@ title: Web Workers 可以使用的函数和类
...
 ## Worker 上下文和函数
 
 **Worker 在另一个全局上下文中运行，与当前的 window 不同！**{{domxref("Window")}} 并不直接在 worker 中可用，其中的很多方法都通过共享的混入（`WindowOrWorkerGlobalScope`）定义，并通过 worker 派生的 {{domxref("WorkerGlobalScope")}} 上下文提供这些方法：
 
-- {{domxref("DedicatedWorkerGlobalScope")}}——专用 worker
-- {{domxref("SharedWorkerGlobalScope")}}——共享 worker
-- {{domxref("ServiceWorkerGlobalScope")}}——[service worker](/zh-CN/docs/Web/API/Service_Worker_API)
+- {{domxref("DedicatedWorkerGlobalScope")}} 用于专用 worker
+- {{domxref("SharedWorkerGlobalScope")}} 用于共享 worker
+- {{domxref("ServiceWorkerGlobalScope")}} 用于 [service workers](/zh-CN/docs/Web/API/Service_Worker_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:16]: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%A4%8D%E6%95%B0%E5%BD%A2%E5%BC%8F

same problem in another PR

```suggestion
- {{domxref("ServiceWorkerGlobalScope")}} 用于 [service worker](/zh-CN/docs/Web/API/Service_Worker_API)
```

Comment (ChaominRuan) [reply files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md]: Have been modified

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 2]
```diff
@@ -3,19 +3,27 @@ title: Web Workers 可以使用的函数和类
...
-一些函数在所有的 worker 和主线程中均可用（来自 `WindowOrWorkerGlobalScope`）：{{domxref("atob", "atob()")}}、{{domxref("btoa", "btoa()")}}、{{domxref("clearInterval", "clearInterval()")}}、{{domxref("clearTimeout()")}}、{{domxref("Window.dump()", "dump()")}} {{non-standard_inline}}、{{domxref("setInterval()")}}、{{domxref("setTimeout()")}}。
+一些函数在所有的 worker 和主线程中均可用（来自 `WindowOrWorkerGlobalScope`）：
+
+- {{domxref("atob", "atob()")}}
+- {{domxref("btoa", "btoa()")}}
+- {{domxref("clearInterval", "clearInterval()")}}
+- {{domxref("clearTimeout()")}}
+- {{domxref("Window.dump()", "dump()")}} {{non-standard_inline}}
+- {{domxref("setInterval()")}}
+- {{domxref("setTimeout()")}}
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:26]: this is not a unordered list in en-US. Could you making a reflect PR in mdn/content?

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 3]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+以下 Web API 在 worker 中可用：
+
+- {{domxref("Barcode_Detection_API","Barcode Detection API")}}
+- {{domxref("Broadcast_Channel_API","Broadcast Channel API")}}
+- {{domxref("Cache", "Cache API")}}
+- {{domxref("Channel_Messaging_API", "Channel Messaging API")}}
+- {{domxref("Console API", "Console API")}}
+- [Web Crypto API](/zh-CN/docs/Web/API/Web_Crypto_API) （例如 {{domxref("Crypto")}}）
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:44]:
```suggestion
- [Web Crypto API](/zh-CN/docs/Web/API/Web_Crypto_API)（例如 {{domxref("Crypto")}}）
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 4]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+- [CSS Font Loading API](/zh-CN/docs/Web/API/CSS_Font_Loading_API)
+- {{domxref("CustomEvent")}}
+- {{domxref("Encoding_API", "Encoding API")}} （例如 {{domxref("TextEncoder")}}，{{domxref("TextDecoder")}}）
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:47]:
```suggestion
- {{domxref("Encoding_API", "Encoding API")}}（例如 {{domxref("TextEncoder")}}、{{domxref("TextDecoder")}}）
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 5]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+- {{domxref("Fetch_API", "Fetch API")}}
+- {{domxref("FileReader")}}
+- {{domxref("FileReaderSync")}} （仅在 worker 中有效！）
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:50]:
```suggestion
- {{domxref("FileReaderSync")}}（仅在 worker 中有效！）
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 6]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+- {{domxref("FormData")}}
+- {{domxref("ImageData")}}
+- {{domxref("IndexedDB_API", "IndexedDB")}}
+- [Network Information API](/zh-CN/docs/Web/API/Network_Information_API)
+- {{domxref("Notifications_API", "Notifications API")}}
+- {{domxref("Performance_API","Performance API")}} ，包括：
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:56]:
```suggestion
- {{domxref("Performance_API","Performance API")}}，包括：
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 7]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+  - {{domxref("Performance")}}
+  - {{domxref("PerformanceEntry")}}
+  - {{domxref("PerformanceMeasure")}}
+  - {{domxref("PerformanceMark")}}
+  - {{domxref("PerformanceObserver")}}
+  - {{domxref("PerformanceResourceTiming")}}
+- {{jsxref("Promise")}}
+- [Server-sent events](/zh-CN/docs/Web/API/Server-sent_events)
+- {{domxref("ServiceWorkerRegistration")}}
+- {{domxref("URL_API","URL API")}} （例如 {{domxref("URL")}}）
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:66]:
```suggestion
- {{domxref("URL_API","URL API")}}（例如 {{domxref("URL")}}）
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 8]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+- {{domxref("Notifications_API", "Notifications API")}}
+- {{domxref("Performance_API","Performance API")}} ，包括：
+  - {{domxref("Performance")}}
+  - {{domxref("PerformanceEntry")}}
+  - {{domxref("PerformanceMeasure")}}
+  - {{domxref("PerformanceMark")}}
+  - {{domxref("PerformanceObserver")}}
+  - {{domxref("PerformanceResourceTiming")}}
+- {{jsxref("Promise")}}
+- [Server-sent events](/zh-CN/docs/Web/API/Server-sent_events)
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:64]:
```suggestion
- [Server-sent 事件](/zh-CN/docs/Web/API/Server-sent_events)
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 9]
```diff
@@ -26,9 +34,46 @@ slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
...
+- [CSS Font Loading API](/zh-CN/docs/Web/API/CSS_Font_Loading_API)
+- {{domxref("CustomEvent")}}
+- {{domxref("Encoding_API", "Encoding API")}} （例如 {{domxref("TextEncoder")}}，{{domxref("TextDecoder")}}）
+- {{domxref("Fetch_API", "Fetch API")}}
+- {{domxref("FileReader")}}
+- {{domxref("FileReaderSync")}} （仅在 worker 中有效！）
+- {{domxref("FormData")}}
+- {{domxref("ImageData")}}
+- {{domxref("IndexedDB_API", "IndexedDB")}}
+- [Network Information API](/zh-CN/docs/Web/API/Network_Information_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:54]: Missing Media Source Extensions API

image

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md (hunk 2 above)
Comment (ChaominRuan) [reply files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:26]: Under review https://github.com/mdn/content/pull/27547

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 10]
```diff
@@ -3,19 +3,31 @@ title: Web Workers 可以使用的函数和类
...
+- {{domxref("DedicatedWorkerGlobalScope")}} 用于专用 worker
+- {{domxref("SharedWorkerGlobalScope")}} 用于共享 worker
+- {{domxref("ServiceWorkerGlobalScope")}} 用于 [service worker](/zh-CN/docs/Web/API/Service_Worker_API)
 
-一些函数在所有的 worker 和主线程中均可用（来自 `WindowOrWorkerGlobalScope`）：{{domxref("atob", "atob()")}}、{{domxref("btoa", "btoa()")}}、{{domxref("clearInterval", "clearInterval()")}}、{{domxref("clearTimeout()")}}、{{domxref("Window.dump()", "dump()")}} {{non-standard_inline}}、{{domxref("setInterval()")}}、{{domxref("setTimeout()")}}。
+一些函数在所有的 worker 和主线程中均可用（来自 `WindowOrWorkerGlobalScope`）：
+
+- {{domxref("atob", "atob()")}}
+- {{domxref("btoa", "btoa()")}}
+- {{domxref("clearInterval", "clearInterval()")}}
```
Comment (yin1999) [inline files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:22]:
```suggestion
- {{domxref("clearInterval()")}}
```

File: files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md (hunk 2 above)
Comment (ChaominRuan) [reply files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md]: > Under review [mdn/content#27547](https://github.com/mdn/content/pull/27547)

@yin1999 mdn/content merged

**Review by yin1999 — APPROVED** [review]
Thanks, but let's simplify a macro call.

**Pull request merged**